sormas-api/src/main/resources/strings.properties

```diff
@@ -791,7 +791,7 @@ infoPickOrCreateImmunization = The system already contains at least one immuniza
 infoPickOrCreateImmunizationExisting = Existing immunization information
 infoPickOrCreateImmunizationNew = Newly added immunization information
 infoPickOrCreatePathogenTest = The database already contains at least one pathogen test that belongs to the sample.<br/><br/>Please look through the lists of pathogen tests. If you feel certain that one matches the lab message details, select it and click on the <i>Confirm</i> button. Otherwise, click on <i>Create new sample</i> to create a new sample for the entry.<br/><br/>If you are unsure, you can discard this window and cancel the process.
-infoPickOrCreateSample = The database already contains at least one sample that seems to be very similar to the details of the lab message.<br/><br/>Please look through the lists of samples. If you feel certain that one matches the lab message details, select it and click on the <i>Confirm</i> button. Otherwise, click on <i>Create new pathogen test</i> to create a new pathogen test for the sample.<br/><br/>If you are unsure, you can discard this window and cancel the process.
+infoPickOrCreateSample = The database does not contain samples that seem to be similar to the details of the lab message. However, the selected entity has other associated samples.<br/><br/>Please look through the list of samples. If you feel certain that one matches the lab message details, select it and click on the Confirm button. Otherwise, click on Create new sample.<br/><br/>If you are unsure, you can discard this window and cancel the process.
```

Comment:
This info does not always fit the situation. It also shows when there is a matching sample:

![image](https://user-images.githubusercontent.com/67001822/169459352-f3e65903-a201-4d50-903b-d7b1ddbfb98d.png)

I propose to either show different messages based on what samples there are, or to generalize the message to fit all situations.

Reply:
It's generic, but I think it could be a little more instructive. How about that?

```
Please choose one of the options below. 
The list of matching samples contains samples in the database that seem to be similar to the details of the lab message (if any).
The list of different samples contains samples related to the entry you chose, but not similar to the details of the lab message (if any).
You can also create a new sample.
```

Reply:
Looks ok to me. But I would keep the also the last sentence: If you are unsure, you can discard this window and cancel the process.

Reply:
Good point.